manifests/server/config.pp

```diff
@@ -49,40 +49,40 @@
         user        => $user,
         auth_method => 'ident',
         auth_option => $local_auth_option,
-        order       => '001',
+        order       => 1,
```

Comment:
Can we change the concat resource to `order => 'numeric'` to help? I think that makes more sense anyway.